src/client.ts

```diff
@@ -123,9 +123,13 @@ export class Client {
   /**
    * Global accessors for the AppSignal Logger API
    */
-  static logger(group: string, level?: LoggerLevel): Logger {
+  static logger(
+    group: string,
+    level: LoggerLevel = "info",
+    format: LoggerFormat = "plaintext"
```

Comment:
We can turn this in a config object if that makes it easier to configure without having to specify defaults only to change the log format. Like keyword arguments in Ruby.

Reply:
I agree, I think we should do this. We need to keep compatibility with the existing signature, though. (It can be done with a [signature overload](https://www.typescriptlang.org/docs/handbook/2/functions.html#overload-signatures-and-the-implementation-signature))

Reply:
I will merge this as is, can you open an issue for this change if you want it?